### PR TITLE
add aimec function for running analysis for just one resType

### PR DESCRIPTION
**Pull request by awirb**
* this can be tested via runAna3AimecIndi.py
* add items to report

### CODE REVIEW

File: avaframe/in3Utils/fileHandlerUtils.py [hunk 1]
```diff
@@ -139,7 +139,7 @@ def extractParameterInfo(avaDir, simName, reportD):
     parameterDict['CPU time [s]'] = stopTime
 
     reportD['Simulation Parameters'].update({'Stop criterion': parameterDict['stop criterion']})
-    reportD['Simulation Parameters'].update({'CPU time [s]': parameterDict['CPU time [s]']})
+    #reportD['Simulation Parameters'].update({'CPU time [s]': parameterDict['CPU time [s]']})
```
Comment (matthiasto) [inline avaframe/in3Utils/fileHandlerUtils.py:142]: remove?

File: avaframe/out3Plot/outAIMEC.py [hunk 2]
```diff
@@ -587,13 +587,20 @@ def resultWrite(cfgPath, cfgSetup, rasterTransfo, resAnalysis):
...
+def resultVisu(cfgPath, cfgFlags, rasterTransfo, resAnalysis, plim, flagIndi=False):
     """
     Visualize results in a nice way
     """
     ####################################
     # Get input data
-    fnames = cfgPath['ppr']
+    if flagIndi:
+        fnames = cfgPath[cfgPath['resType']]
+        maxMaxDPPR = resAnalysis['MMP']
```
Comment (matthiasto) [inline avaframe/out3Plot/outAIMEC.py:598]: why not save it as MMPPR to avoid the if and the flag?

File: avaframe/ana3AIMEC/ana3AIMEC.py [hunk 3]
```diff
@@ -360,7 +444,8 @@ def postProcessAIMEC(rasterTransfo, pLim, newRasters, cfgPath, cfgFlags):
 
     return resAnalysis
 
-def postProcessAIMECReport(rasterTransfo, pLim, newRasters, cfgPath, cfgFlags):
+
+def postProcessAIMECReport(rasterTransfo, newRasters, cfgSetup, cfgPath, cfgFlags):
```
Comment (fso42) [inline avaframe/ana3AIMEC/ana3AIMEC.py:448]: What ist the difference of this function to `postProcessAIMEC` ? There seems to be a lot of code overlap.

File: avaframe/ana3AIMEC/ana3AIMECCfg.ini [hunk 4]
```diff
@@ -19,15 +12,29 @@ domainWidth = 600
...
+# max of resType difference for contour lines plus capped difference in resType plot (for example 1 (pfd), 5 (ppr))
+diffLim = 1
 
 # chose interpolation method between 'nearest' and 'bilinear'
 interpMethod = bilinear
 
 # resampling step [m]
 distance = 10
 
+# computational module used for analysing the results of
```
Comment (fso42) [inline avaframe/ana3AIMEC/ana3AIMECCfg.ini:34]: Can we write this in a clearer way? Especially in comparison to `comModules` below? It is not clear what the difference is. Maybe something like: 'which computational module is analysed` or `Target module for the analysis`....

**Review by fso42 — CHANGES_REQUESTED** [review]

File: avaframe/out3Plot/outAIMEC.py [hunk 5]
```diff
@@ -266,6 +266,10 @@ def visuMass(resAnalysis, cfgPath, cfgFlags):
     DataMass = np.array(([None] * 2))
     DataMass[0] = entMassArray
     DataMass[1] = totalMassArray
+    print('Data here', entMassArray)
+    print('Data here', type(DataMass))
+    print('Data here', DataMass[0])
+    print('Data here', DataMass[1])
```
Comment (matthiasto) [inline avaframe/out3Plot/outAIMEC.py]: did you forget to remove this?

File: avaframe/ana3AIMEC/ana3AIMEC.py [hunk 6]
```diff
@@ -479,25 +466,12 @@ def postProcessAIMECReport(rasterTransfo, newRasters, cfgSetup, cfgPath, cfgFlag
...
-                                  'aCrossMax' : PFDCrossMax,
-                                  'aCrossMean' : PFDCrossMean}
-    resultsAreaAnalysis['pfv'] = {'transformedRasters' : dataSpeed,
-                                  'maxaCrossMax' : maxPFVCrossMax,
-                                  'aCrossMax' : PFVCrossMax,
-                                  'aCrossMean' : PFVCrossMean}
+    # get max and mean values along path for cross profiles
+    resultsAreaAnalysis = aT.analyzeField(rasterTransfo, dataPressure, 'ppr', resultsAreaAnalysis)
+    resultsAreaAnalysis = aT.analyzeField(rasterTransfo, dataDepth, 'pfd', resultsAreaAnalysis)
+    resultsAreaAnalysis = aT.analyzeField(rasterTransfo, dataSpeed, 'pfv', resultsAreaAnalysis)
```
Comment (matthiasto) [inline avaframe/ana3AIMEC/ana3AIMEC.py:474]: why did you not use the same method as in postProcessAIMEC ?

File: avaframe/out3Plot/outAIMEC.py (hunk 5 above)
Comment (awirb) [reply avaframe/out3Plot/outAIMEC.py]: this is not in the recent version, no?

File: avaframe/ana3AIMEC/ana3AIMECCfg.ini (hunk 4 above)
Comment (awirb) [reply avaframe/ana3AIMEC/ana3AIMECCfg.ini]: done

**Review by fso42 — APPROVED** [review]

**Pull request merged**